build.sbt

```diff
@@ -0,0 +1,20 @@
+ThisBuild / organization := "com.bedrocksreaming.data"
+ThisBuild / version := "0.1.0"
+ThisBuild / scalaVersion := "2.12.11"
+
+// ********
+// Versions
+// ********
+
+val SparkVersion = "3.0.3"
```

Comment:
We don't use the latest Spark version. Is it on purpose? If yes, I think we should precise it.

Reply:
I think the best thing to do is to deploy the lib for each spark minor version
I am afraid that if we deploy it with `Spark 3.3` it can cause issues with projects using `3.0`? Don't you think?